ush/ioda/bufr2ioda/run_bufr2ioda.py

```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env python3
+import argparse
+import os
+from gen_bufr2ioda_json import gen_bufr_json
+from wxflow import Logger, Executable, cast_as_dtype
+
+# Initialize root logger
+logger = Logger('run_bufr2ioda.py', level='INFO', colored_log=True)
+
+
+def bufr2ioda(CDATE, RUN, DMPDIR, config_template_dir, COM_OBS):
+    logger.info(f"Process {CDATE} {RUN} from {DMPDIR} to {COM_OBS} using {config_template_dir}")
```

Comment:
```suggestion
@logit(logger)
def bufr2ioda(CDATE, RUN, DMPDIR, config_template_dir, COM_OBS):
    logger.info(f"Process {CDATE} {RUN} from {DMPDIR} to {COM_OBS} using {config_template_dir}")
```

Reply:
Suggestion accepted at cff2ce5.  Question:  why do we add `@logit(logger)`?

Reply:
`logit` is a decorator.  Its use is to log the input and output of the function it decorates.
Adding that one liner, will yield to the user, what are the inputs being sent to the function and what the outputs are being returned from the function.  Additionally, it also logs the entry and exit times that can be useful for profiling.
If any additional logging or profiling stats (e.g. memory) are desired, they can be implemented in `logit` and the entire project can reap the benefits.

Reply:
Thank you @aerorahul for explaining what `@logit` brings to the table.